gui-wm/river/river-0.3.5-r1.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 2021-2022 Aisha Tammy
+# Distributed under the terms of the ISC License
+
+EAPI=8
+
+inherit edo
+
+DESCRIPTION="Dynamic tiling wayland compositor"
+HOMEPAGE="https://codeberg.org/river/river"
+
+SRC_URI="
+https://codeberg.org/river/river/releases/download/v${PV}/${P}.tar.gz
+https://codeberg.org/ifreund/zig-pixman/archive/v0.2.0.tar.gz -> zig-pixman-0.2.0.tar.gz
+https://codeberg.org/ifreund/zig-wayland/archive/v0.2.0.tar.gz -> zig-wayland-0.2.0.tar.gz
+https://codeberg.org/ifreund/zig-wlroots/archive/v0.18.0.tar.gz -> zig-wlroots-0.18.0.tar.gz
+https://codeberg.org/ifreund/zig-xkbcommon/archive/v0.2.0.tar.gz -> zig-xkbcommon-0.2.0.tar.gz
+"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+
+IUSE="+man pie test +X no-llvm"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-libs/libevdev
+	dev-libs/libinput
+	dev-libs/wayland
+	>=gui-libs/wlroots-0.18.0:=[X?]
+	x11-libs/libxkbcommon:=[X?]
+	x11-libs/pixman
+"
+DEPEND="${RDEPEND}"
+EZIG_VISION="0.13*"
+BDEPEND="
+	|| ( =dev-lang/zig-${EZIG_VISION} =dev-lang/zig-bin-${EZIG_VISION} )
+	dev-libs/wayland-protocols
+	man? ( app-text/scdoc )
+	sys-apps/coreutils:=[-static]
```

Comment:
```suggestion
	sys-apps/coreutils[-static]
```
Like this

Reply:
Ah i see, good to know, i just used that because it was the first example that i saw in another package lol, im commiting your change now